### PR TITLE
Add theadactyl as community manager

### PR DESCRIPTION
**Pull request by theadactyl**
Added Community Manager role to Org.Operators and added theadactyl as community manager

### CODE REVIEW

**Comment by tianon** [comment]
Next time, please clone, amend, and force push your commit like @GordonTheTurtle suggested (https://github.com/docker/docker/pull/13370#issuecomment-104069136) instead of opening a new PR. :wink:

**Comment by crosbymichael** [comment]
no one listens to bots

**Comment by jfrazelle** [comment]
the bots are here so i have friends, that is their sole purpose, LGTM

**Comment by theadactyl** [comment]
thanks @tianon. had some git configuration to work through and got lazy. (shame)

**Comment by crosbymichael** [comment]
@theadactyl  first rule of community management, if something goes wrong, blame @tianon

File: MAINTAINERS [hunk 1]
```diff
@@ -319,6 +324,17 @@ made through a pull request.
 	# day of a new maintainer, the best advice should be "follow the C.M.'s example and you'll
 	# be fine".
 	"Chief Maintainer" = "crosbymichael"
+	
+	# The community manager is responsible for serving the project community, including users, 
+	# contributors and partners. This involves:
```
Comment (thaJeztah) [inline MAINTAINERS]: Yup, nitpicking again; indentation of these lines and below looks "off"

**Comment by thaJeztah** [comment]
..all [those docs](https://docs.docker.com/project/create-pr/) go unread 😢

added a note, other than that LGTM. Welcome Thea!

**Comment by theadactyl** [comment]
@thaJeztah can you point out where the indentation is funky? I'm not seeing it :(

**Comment by thaJeztah** [comment]
@theadactyl the first two lines of the comment block looked to be indented 1 space less than the rest. But my comment now makes that hard to see. 😇

I'm on mobile now, so hard to be sure

it's really a nit, so no worries if it stays in for now. Getting this merged is the important part :)

**Comment by thaJeztah** [comment]
🎉

**Pull request merged**